docs/metrics/work-in-progress.md

```diff
@@ -0,0 +1,20 @@
+# Work in progress metrics
+
+## Work in progress
+
+To provide you with an high-level view of the current work in progress of your teams, Pulse groups pull requests that are currently open by the following phases:
+
+-   **Open:** the pull request is open but the team hasn't interacted or reviewed it.
+-   **In review:** the pull request received at least one review but it isn't approved yet. Reviews include approvals, change requests, and inline pull request comments, but don't include conversation comments.
+-   **Approved:** the pull request received at least one approval.
+
+## Work in progress details
+
+The following metrics for each open pull request allow you to understand which work items are about to become or have become blockers and may need an action to move forward:
+
+-   **Time in progress:** number of days that the pull request is in progress.
+-   **Phase:** current phase of the pull request, can be one of **Open**, **In review**, or **Approved**.
```

Comment:
```suggestion
-   **Phase:** current phase of the pull request, can be one of **Open**, **Reviewed**, or **Approved**.
```

I think we had always talked about `Reviewed` instead of `In review` and it sounds more appropriate since usually a review means it is reviewed.

Reply:
I'm leaving this conversation open for now so that @claudiacodacy can also see the suggestion and perhaps reflect the change on [Figma](https://www.figma.com/file/4AK8l4ThiOphTaqFNuPhbq/Pulse?node-id=638%3A8293).

Reply:
> I think we had always talked about Reviewed instead of In review and it sounds more appropriate since usually a review means it is reviewed

I initially proposed "In review" because I was thinking about it like a status that is in progress, whereas "reviewed" felt like it had been reviewed and that was it. I don't have a strong opinion about it, in truth all other states use the past-tense, so that would make them more consistent. I'll change them, and we can confirm this later :)

Reply:
Thank you for your input @claudiacodacy! Personally, I think that **Reviewed** is a little simpler. :+1: